docs/general/ddl/data-types.rst

```diff
@@ -437,6 +443,69 @@ In these expressions, the desired time zone is specified as a string
 The scalar function :ref:`TIMEZONE <scalar-timezone>` (zone, timestamp) is
 equivalent to the SQL-conforming construct timestamp AT TIME ZONE zone.
 
+.. _time-data-type:
+
+time with time zone
```

Comment:
Time with time zone

or

Time with Time Zone

?

Reply:
in this case I would keep lowercase, as it is how you normally would spell it on the crate shell.